### PR TITLE
Add CoC Id translation

### PR DESCRIPTION
**Pull request by iynsys**
## What does this PR do?
Add CoC indonesian

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LuigiImVector** [comment]
New course isn't in alphabetical order

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -418,6 +418,7 @@
 * [learn:query](https://learnquery.infinum.co)
 * [Namaste 🙏 JavaScript: An In-Depth JavaScript Fundamentals Course](https://www.youtube.com/playlist?list=PLlasXeu85E9cQ32gLCvAvr9vNaUccPVNP) - Akshay Saini (YouTube)
 * [Programming Foundations with Javascript, HTML and CSS](https://www.coursera.org/learn/duke-programming-web) - Owen Astrachan, Robert Duvall, Andrew D. Hilton, Susan H. Rodger (Coursera)
+* [Functional Programming in Javascript](https://www.youtube.com/playlist?list=PL0zVEGEvSaeEd9hlmCXrk5yUyqUag-n84) - MPJ aka Fun Fun Function(Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:421]: Please, use right creator attribution and leave a whitespace between each resource field

Use this snippet:
```markdown
* [Functional Programming in Javascript](https://www.youtube.com/playlist?list=PL0zVEGEvSaeEd9hlmCXrk5yUyqUag-n84) - Mattias Petter Johansson «Fun Fun Function» (Youtube)
```

And to solve list alphabetization move it to line 410

**Review by davorpa — CHANGES_REQUESTED** [review]
Ensure that linter rules are preserved

```logs
Run fpb-lint ./courses/
courses/free-courses-en.md
  406:1-421:151  warning  Alphabetical ordering: swap l.421 and l.420  alphabetize-lists  remark-lint

⚠ 1 warning
Error: Process completed with exit code 1.
```

File: CODE_OF_CONDUCT-id.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Contoh perilaku yang kasar, melecehkan, atau tidak dapat diterima mungkin
+dilaporkan dengan menghubungi pengelola proyek di victorfelder di gmail.com. Semua
+keluhan akan ditinjau dan diselidiki dan akan menghasilkan tanggapan yang
+dianggap perlu dan sesuai dengan keadaan. Pengelola adalah
+wajib menjaga kerahasiaan terhadap pelapor suatu
+insiden.
+
+
+Kode Etik ini diadaptasi dari [Contributor Covenant][homepage],
+versi 1.3.0, avaible at https://contributor-covenant.org/version/1/3/0/
```
[the remark this answers is not in the document]
Comment (iynsys) [reply CODE_OF_CONDUCT-id.md:45]: Thanks for reply, Im first timer

**Review by iynsys — COMMENTED** [review]
Is this right? 
Thank you for review

**Review by iynsys — COMMENTED** [review]
Thank you
Please fix me if i do wrong

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -420,6 +421,7 @@
 * [Programming Foundations with Javascript, HTML and CSS](https://www.coursera.org/learn/duke-programming-web) - Owen Astrachan, Robert Duvall, Andrew D. Hilton, Susan H. Rodger (Coursera)
 
 
+
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:424]:
```suggestion
```

You just added one too many black lines.

**Review by iynsys — COMMENTED** [review]
Thank for review 
I've already delete line 424

File: courses/free-courses-en.md (hunk 3 above)
Comment (iynsys) [reply courses/free-courses-en.md]: Thanks 
I've already fix that

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -407,6 +407,7 @@
 * [Bento JavaScript Learning Track](https://bento.io/topic/javascript) (Bento)
 * [Egghead.io](https://egghead.io)
 * [ES6 and Typescript Tutorial](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhI0V5mE9Vu6Nm-nap8EcjV) - Codevolution, Vishwas Gopinath (YouTube)
+* [Functional Programming in Javascript](https://www.youtube.com/playlist?list=PL0zVEGEvSaeEd9hlmCXrk5yUyqUag-n84) - Mattias Petter Johansson «Fun Fun Function» (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:410]: Some usefull notes... next time try to fit to task :wink: (aka translate COC)

**Review by davorpa — APPROVED** [review]
**GREATTTT CHAMPION!! You get it!!! Big effort together**

### Ready to merge :fireworks: :tada:

File: courses/free-courses-en.md (hunk 4 above)
Comment (iynsys) [reply courses/free-courses-en.md:410]: Alright.. thanks for correcting me 😄

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by iynsys** [comment]
> Thanks, and welcome to free-programming-books!

Can i do more PRs on this repo? 😃

**Pull request merged**